utils/controller.js

```diff
@@ -536,33 +536,41 @@ class ABServiceController extends EventEmitter {
                      }
                   }
 
+                  const pool = workerpool.pool();
```

Comment:
This will create a new pool for each request. Would it be better to reuse a common pool? I'm not sure how well it will work with multiple requests at the same time.

Reply:
Yes, it is still better to reuse even though we have many multiple requests concurrently.
https://github.com/digi-serve/ab-utils/pull/62/commits/bc27e6bfe9a5ae9adcaa910019acf7d2170b6025